sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/InsertIntoHadoopFsRelationCommand.scala

```diff
@@ -148,6 +148,9 @@ case class InsertIntoHadoopFsRelationCommand(
         options = options)
 
       fileIndex.foreach(_.refresh())
+      catalogTable.foreach { table =>
+        sparkSession.sharedState.cacheManager.uncacheQuery(sparkSession.table(table.identifier))
```

Comment:
it is recached? https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala#L154